src/Options/JobOptions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace SlmQueueBeanstalkd\Options;
+
+use Laminas\Stdlib\AbstractOptions;
+use Pheanstalk\Pheanstalk;
+
+class JobOptions extends AbstractOptions {
+    protected $__strictMode__ = false;
```

Comment:
What does strict mode do? And why is it wrapped around with `__`? Please do add a comment about this.